src/replaykit.cs

```diff
@@ -259,5 +259,9 @@ interface RPBroadcastSampleHandler {
 
 		[Export ("processSampleBuffer:withType:")]
 		void ProcessSampleBuffer (CMSampleBuffer sampleBuffer, RPSampleBufferType sampleBufferType);
+
+		[iOS (10,2)][TV (10,1)]
+		[Export ("finishBroadcastWithError:")]
+		void FinishBroadcast (NSError error);
```

Comment:
also intro signature check should spot that... at least for a type but it would not for delegates so it's good to comment on them :)